content/modules/r/pages/apply-functions.adoc

```diff
@@ -0,0 +1,411 @@
+= Apply Functions
+
+== 'apply'
+
+== 'lapply'
+
+The 'lapply' is a function that applies a function 'FUN' to each element in a vector or list, and returns a list.
+
+== Examples
```

Comment:
Let's nest the examples in each section under each section. So make it:

```asciidoc
=== Examples
```

And then each following example make a level 4 header:

```asciidoc
==== Question text here
```

Thanks!